File: src/domain/models/WeatherDataModel.py
from dataclasses import dataclass

@dataclass
class WeatherDataModel:
    weather_station_id: int
    date: str
    WW: str
    T: float
    Ff: float
    P: float
    U: float
    VV: float
    Td: float
    RRR: float
    WW_code: int
    lon: float
    lat: float
    WW_type: str

File: Domain/Models/WeatherDataModel.cs
namespace FireDataWebService.Domain.Models
{
    public class WeatherDataModel
    {
        public int WeatherStationId { get; set; }
        public DateTime Date { get; set; }
        public string? WW { get; set; }
        public double? T { get; set; }
        public double? Ff { get; set; }
        public double? P { get; set; }
        public double? U { get; set; }
        public double? V { get; set; }
        public double? VV { get; set; }
        public double? Td { get; set; }
        public double? RRR { get; set; }
        public double? WWCode { get; set; }
        public double? Lon { get; set; }
        public double? Lat { get; set; }
        public string? WWType { get; set; }
    }
}
